release_notes/ose_3_1_release_notes.adoc

```diff
@@ -160,7 +164,22 @@ command to SSH into privileged containers.
 * https://bugzilla.redhat.com/show_bug.cgi?id=1267670[BZ 1267670]: There was an
 issue with using the `iptables` command with the `-w` option to make the
 `iptables` command wait to acquire the *xtables* lock, causing some SDN
-initializations to fail. This issue has now been fixed.
+initializations to fail. This issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1272201[BZ 1272201]: When using two network interfaces, installing a clustered etcd and defining variables for IP and etcd interfaces, the certificate would be populated with only the first
+network, instead of whichever network was desired. The issue has now been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1269256[BZ 1269256]: Using the `GET` fieldselector would return a 500 BadRequest error. This issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1268000[BZ 1268000]: Previously, creating an application from a imagestream could result in two builds being initiated. This was caused by the wrong imagestream tag being used by the build process. The issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1267231[BZ 1267231]: The *ose-ha-proxy* router image was missing the x-forwarded header, causing the Jenkins application to redirect to HTTP instead of HTTPS. The issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1276548[BZ 1276548]: Previously, an error was present where the HAProxy router did not expose stats, even if the port was specified. The issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1275388[BZ 1275388]: Previously, some node hosts would not talk to the SDN due to routing table differences. A `*lbr0*` entry was causing traffic to be routed incorrectly. The issue has been fixed.
+* https://bugzilla.redhat.com/show_bug.cgi?id=1265187[BZ 1265187]: When persistent volume claims were created from a template, sometimes the same volume would be mounted to multiple persistent volume claims. At the same time, the volume would show that only one PVC was being used. The issue has been fixed.
```

Comment:
"persistent volume claims (PVCs)" for the first one, then PVC for the two after?